tinymce-angular-component/src/TinyMCE.ts

```diff
@@ -6,7 +6,7 @@
  *
  */
 
-declare let global: any;
+let global: any;
```

Comment:
I'm not sure this is correct... When using declare, the ```ng serve``` command didn't work anymore